.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Existing tag to be released, e.g. v1.2.3, v2.2.2, v3.0.0'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Deploy release
+    steps:
+      - name: Determine tag to be released
```

Comment:
this step doesn't determine but only sets vars. Worth renaming?

Reply:
I think the name is kinda good, we are extracting tag and version from GITHUB_REF or from the user input. Will rename it to "extract tag and version to be released"

Reply:
sounds good. thanks

Reply:
done in https://github.com/hazelcast/hazelcast-wm/pull/137/commits/6846f07329d967741694291026419f10a325a822

Reply:
thanks